### ISSUE TITLE
Prepare ESO chart GS Repo

### ISSUE THREAD

**Issue by gianfranco-l** [giantswarm/roadmap#1767]
- [x] fork the [upstream Helm chart](https://github.com/external-secrets/external-secrets/tree/main/deploy/charts/external-secrets)
    - [x] fix problem: our kyverno validation rules require that `app.kubernetes.io/version` label is part of a pod selector definition in `Deployments`; it's missing, so the validation fails
    - [x] figure out the default `values.yaml` file ([see the spike ticket](https://github.com/giantswarm/giantswarm/issues/24452#issuecomment-1328973537))
    - [x] PSP - ensure `projected` volumes are enabled in the chart
- [x] add `abs`/`ats` config
- [x] add at least basic tests for `ats`

**Comment by uvegla** [comment]
The `app.kubernetes.io/version` is now added in `v0.7.0` automatically (well if the `.Chart.AppVersion` is set), so ticked that off as not actionable by now.

**Comment by uvegla** [comment]
I am not sure about `PSP - ensure projected volumes are enabled in the chart`, had no issues with it so far. 🤔

**Comment by uvegla** [comment]
FTR in my experience it takes ~90-120 seconds for the cert controller and the webhook pod to be ready.

**Comment by uvegla** [comment]
Merged: https://github.com/giantswarm/eso/pull/1 and https://github.com/giantswarm/github/pull/736.

FTR it has to be added back to `retagger` too when Aliyun gets unblocked.

**Comment by uvegla** [comment]
Split the `retagger` part into a separate ticket: https://github.com/giantswarm/giantswarm/issues/25132. Closing this one as we have a working chart.
